packages/donations-app/app/components/auth/registrationModal.html

```diff
@@ -65,48 +68,56 @@ <h2 class="modal-title">Sign up</h2>
 
         <div class="row"
              ng-show="regCtrl.duplicatedEmail && (regCtrl.duplicatedEmail == regCtrl.registration.email)">
-          <div class="col-md-12">
-            <p class="text-error">
-              User with email {{regCtrl.duplicatedEmail}} is already registered
-            </p>
-          </div>
+          <p class="text-error col-md-12">
+            User with email {{regCtrl.duplicatedEmail}} is already registered
+          </p>
         </div>
 
-        <div class="registration-terms col-md-12">
-          By tapping Sign Up you agree to our
-          <a href="https://s3.eu-west-2.amazonaws.com/alice-res/terms_of_use.pdf"
-             target="_blank">
-             Terms & Conditions
-           </a>
-             and our
-          <a href="https://s3.eu-west-2.amazonaws.com/alice-res/privacy_policy.pdf"
-             target="_blank">
-             Privacy Policy
-           </a>
+        <div class="row">
+          <p class="col-md-12 alice-grey center">
+            By tapping Sign Up you agree to our
+            <a href="https://s3.eu-west-2.amazonaws.com/alice-res/terms_of_use.pdf"
+               target="_blank">
+               Terms & Conditions
+            </a> and our
+            <a href="https://s3.eu-west-2.amazonaws.com/alice-res/privacy_policy.pdf"
+               target="_blank">
+               Privacy Policy
+            </a>
+          </p>
         </div>
-        <button id="registration-sign-up-button"
-                class="registration-sign-up btn btn-primary btn-donate btn-modal"
-                ng-click="regCtrl.register()"
-                type="submit">
-                Sign Up
-        </button>
-        <div class="registration-agreement col-md-12">
-          <div class="registration-agreement-content">
-            <input icheck
-                   type="checkbox"
-                   ng-model="regCtrl.registration.disagreeContact">
-               I don't want to receive marketing messages <br>from Alice or the charity.
+
+        <div class="row">
```

Comment:
I suppose I makes sense to offer a link for a simplified "e-mail" only sign-up from a full sign-up. What do you think?